### PR TITLE
Add Lyles Sutherland case study

### PR DESCRIPTION
**Pull request by Ken-Lo**
screen shot 2016-07-25 at 12 06 02

### CODE REVIEW

**Comment by Ken-Lo** [comment]
Hey @sjnorth 
Updated case study with images. Let's see if we want to update the header or add additional images, and then we can merge it in afterwards :)

File: app/pages/stories/stories/lyles-sutherland.js [hunk 1]
```diff
@@ -1,95 +1,125 @@
...
-        sidebarLogoSrc='/images/stories/story-logos/hotpod-yoga.png'
-        sidebarServices='Providing hot yoga to the masses'
-        sidebarEmployees='2 employees, 16 franchisees and 45 instructors.'
-        sidebarPreviousProvider='Manual invoices'
-        sidebarTimeSaved='Currently 1-2 hours per month but this time is growing rapidly as they scale up.'>
+        sidebarLogoSrc='/images/stories/story-logos/lyles-sutherland.png'
+        sidebarServices='Digital agency and software developer'
+        sidebarEmployees='16 employees'
+        sidebarPreviousProvider='New to Direct Debit'
+        sidebarTimeSaved='A couple of hours a week.'>
```
Comment (jacobpgn) [inline app/pages/stories/stories/lyles-sutherland.js]: This line has a full-stop, but the others don't. Looks a little odd?

Comment (Ken-Lo) [inline app/pages/stories/stories/lyles-sutherland.js]: Good spot - thanks Jacob - have just fixed this!

**Comment by sjnorth** [comment]
I have added the changes from the client - this is now ready to go @Ken-Lo

**Comment by Ken-Lo** [comment]
@sjnorth +1 Great work!

**Pull request merged**